recipes/flang-activation/conda_build_config.yaml

```diff
@@ -0,0 +1,29 @@
+CBUILD:
+  - x86_64-conda-linux-gnu      # [linux]
+# - x86_64-conda-linux-gnu      # [linux]
+  - x86_64-pc-windows-msvc      # [win]
+  - dummy                       # [osx]
+
+CHOST:
+  - x86_64-conda-linux-gnu      # [linux]
+# - wasm32-unknown-emscripten   # [linux]
+  - x86_64-pc-windows-msvc      # [win]
+  - dummy                       # [osx]
+
+cross_target_platform:
+  - linux-64                    # [linux]
+# - emscripten-wasm32           # [linux]
+  - win-64                      # [win]
+  - dummy                       # [osx]
+
+zip_keys:
+  -
+    - CBUILD
```

Comment:
This shouldn't be in the zip_keys with the others. Not necessary for now since there's no cross-compilation, but will be useful in the future,